dspace-server-webapp/src/test/java/org/dspace/app/rest/SearchEventRestRepositoryIT.java

```diff
@@ -411,4 +411,59 @@ public void postTestSuccesEmptyQuery() throws Exception {
                    .andExpect(status().isCreated());
 
     }
+
+    @Test
+    public void postTestWithObjectSuccess() throws Exception {
+
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and two collections.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                .withName("Parent Community")
+                .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                .withName("Sub Community")
+                .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+
+        //2. Three public items that are readable by Anonymous with different subjects
+        Item publicItem1 = ItemBuilder.createItem(context, col1)
+                .withTitle("Public item 1")
+                .withIssueDate("2017-10-17")
+                .withAuthor("Smith, Donald").withAuthor("Doe, John")
+                .withSubject("ExtraEntry")
+                .build();
+
+        context.restoreAuthSystemState();
+
+        SearchEventRest searchEventRest = new SearchEventRest();
+
+        searchEventRest.setQuery("test");
+        searchEventRest.setScope(publicItem1.getID());
+        searchEventRest.setConfiguration("default");
+        searchEventRest.setDsoType("item");
+        searchEventRest.setObject(publicItem1.getID());
```

Comment:
I've modified `setScope()` to set it to the item's collection instead, to make more sense.
`setObject()` has also been renamed to `setClickedObject()`, so that should clear things up even more.